pkg/util/kubeconfig.go

```diff
@@ -344,3 +344,29 @@ func SetCurrentContext(kubeCfgPath, name string) error {
 	}
 	return nil
 }
+
+// DeleteKubeConfig deletes the specified machine's kubeconfig file
+func DeleteKubeConfig(kubeCfgPath, machineName string) error {
```

Comment:
I think it is better to rename this to DeleteKubeContext, since we are not actually deleting kubeconfig file.

Reply:
I agree with you. I am gonna update! Thank you for your suggestion.

Reply:
@medyagh Updated. PTAL.